euphonic/readers/phonopy.py

```diff
@@ -181,36 +195,55 @@ def _extract_band_data_hdf5(hdf5_file):
 
 
 def _read_phonon_data(
-        path='.', phonon_name='band.yaml', phonon_format=None,
-        summary_name='phonopy.yaml', cell_vectors_unit='angstrom',
-        atom_mass_unit='amu', frequencies_unit='meV'):
+        path: Optional[str] = '.', phonon_name: Optional[str] = 'band.yaml',
```

Comment:
Again, I'm not sure that all of these should have the `Optional` type, which is really a shorthand for `Union[None, type]`. Arguments with default values can just be set as e.g. `path: str = '.'`.